style.css

```diff
@@ -0,0 +1,3 @@
+body{
```

Comment:
please remember to apply the appropriate line spacing in your css

Reply:
Kindly add an explanation of why you requested the review. Remember you are trying to correct here and you want the person you are correcting to understand why you made a particular request.